experimental/dds/tree2/src/core/schema-stored/builders.ts

```diff
@@ -49,7 +49,7 @@ export function fieldSchema(
  */
 export interface TreeSchemaBuilder {
 	readonly structFields?: { [key: string]: FieldStoredSchema };
-	readonly mapFields: FieldStoredSchema;
+	readonly mapFields?: FieldStoredSchema;
 	readonly leafValue?: ValueSchema;
```

Comment:
Can these inherit the docs from TreeStoredSchema?

Reply:
they types are not compatible. There are different representations of the same information so neither can extend each other. That said, this type has no reason to be in the public package API anymore, so I've fixed that. Now it's less-than-ideal docs shouldn't matter much (its used in ~2 places internally).

Reply:
I guess {@inheritdoc} is an option. I'm not going to bother for now since this type is now internal, and likley to be removed soon, but thats an option I might want to use elsewhere.